### DISCUSSION TITLE
Need help  for the Pull Request task

### DISCUSSION THREAD

**Discussion by VictoriAragon** [earthlab-education/Earth-Analytics-AY24#109] [General]
Maybe a quick walk through in the next class if there is time.

**Comment by eculler** [comment]
Hi @VictoriAragon -- we'll need a little more information to help. Can you explain:

1. What you are trying to do
2. What you did
3. What actually happened

Be sure to include the text of any error messages or screenshots to help us help you!

P.S. You can definitely bring up your issue in class, AND learning to write bug reports like this is something I hope you'll get out of this course! We'll be talking more about how to do that for sure. I find that about 90% of the time when I start writing a bug report I solve my issue before I'm done.

**Comment by nquarder** [comment]
Hi @VictoriAragon - yes we can do a demonstration of how to submit a Pull Request in today's discussion. See you then! -NQ
